bobber/bobber.py

```diff
@@ -131,6 +133,16 @@ def parse_args(version: str) -> Namespace:
                                  type=int)
     commands_parent.add_argument('--iops-threads', help='Maximum number of '
                                  'threads to use for iops tests', type=int)
+    commands_parent.add_argument('--read-pattern', help='Specify IO pattern '
+                                 'for fio read tests. Supported values: '
+                                 'sequential, random. Defaults to sequential.',
+                                 default='sequential',
+                                 choices=READ_PATTERNS.keys())
+    commands_parent.add_argument('--write-pattern', help='Specify IO pattern '
+                                 'for fio write tests. Supported values: '
+                                 'sequential, random. Defaults to sequential.',
+                                 default='sequential',
+                                 choices=WRITE_PATTERNS.keys())             
```

Comment:
Looks like there's some extra whitespace at the end of this line.

Reply:
Yeah I cleaned that up...was weirdly misbehaving (or not behaving as I expected, anyway) in VS Code.